examples/README.md

```diff
@@ -1,37 +1,39 @@
-
 # Examples
 
-The examples in this directory are applications that can be run against your cluster.
+The examples in this directory are applications that can be run on your Mantl
+cluster.
 
-In each example directory is a README.md that provides the short and sweet instructions. Below we go through
-the examples in more detail and provide more of an introduction to what is going on.
+In each example directory is a README.md that provides the short and sweet
+instructions. Below we go through the examples in more detail and provide more
+of an introduction to what is going on.
 
 ## Getting a Cluster Up
 
-#### Vagrant Cluster
-
-Getting the vagrant cluster up and running on your laptop is the easiest way to get a cluster going
-for the examples.
+#### Vagrant
 
-To do this, see Getting Started in the [README.md](../README.md) at root of this project.
+Getting the vagrant cluster up and running on your computer is the easiest way
+to get a cluster going for the examples. To do this, see Getting Started in the
+[README.md](../README.md) at root of this project.
 
-If you have not done so yet, go ahead get vagrant up now. It will take a couple of minutes and you can read
-on while you wait.
+If you have not done so yet, go ahead get vagrant up now. It will take a couple
+of minutes and you can read on while you wait.
 
 To run these examples you will need to know:
 
 1. The url for marathon:
-Marathon runs on port 8080 and if you look in the Vagrantfile in the root of this project there is a line:
-
-        VAGRANT_PRIVATE_IP = "192.168.242.55"
+Marathon runs on port 8080 and if you look in
```

Comment:
The marathon api is also available on control nodes over http(s) at the /marathon endpoint. Should we consider updating the examples to use `https://192.168.100.101/marathon/v2/apps` instead?